saskatoon/harvest/api.py

```diff
@@ -223,3 +223,19 @@ def list(self, request, *args, **kwargs):
                                  'title': _("New Person")
                                  }
                          })
+
+class HarvestYieldViewset(LoginRequiredMixin, viewsets.ModelViewSet):
+    template_name = 'app/stats.html'
+    queryset = HarvestYield.objects.all().order_by('-id')
+    serializer_class = HarvestYieldSerializer
+    filter_backends = (filters.DjangoFilterBackend,)
+    filterset_class = HarvestYieldFilter
+    filterset_fields = '__all__'
+
```

Comment:
@mina-andajos ideally we should enforce custom permissions on this view, e.g. `permission_classes = [IsCoreOrAdmin]`
https://www.django-rest-framework.org/api-guide/permissions/#custom-permissions 
The `IsCoreOrAdmin` permission should subclass `IsAuthenticated` and make sure that the `request.user` belongs to either the `core` or `admin` group ( e.g. something like `request.user.groups.filter(name__in=["core", "admin"]).exists()` )
Let me know if you feel like implementing this, otherwise I can do it when you're done with you PR.
Thanks!

Reply:
Thanks for pointing this out. I'll be implementing it.

Reply:
Had to create a `core` and an `admin` group since they didn't exist for me.

------------------------
### Screenshots:
1- not and in admin or core
![image](https://user-images.githubusercontent.com/52796958/161556152-5d163b39-5af7-4963-aaaa-c7d4f18347a8.png)

2- core only
![image](https://user-images.githubusercontent.com/52796958/161556289-a5201724-8192-46ec-9864-b74147eefa52.png)

3- admin & core
![image](https://user-images.githubusercontent.com/52796958/161556397-607571fc-e768-4e47-8bdf-fbe7696f2b71.png)

Reply:
yea sorry the groups are loadded with `python saskatoon/manage.py saskatoon/fixtures/auth-group.json`